packages/core/template/project/AboutUs.md

```diff
@@ -0,0 +1,61 @@
+---
+  layout: default.md
+  title: "About Us"
+---
+
+# About Us
+
+We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
```

Comment:
Would probably swap this out for more dummy content (particularly the email, if it's a real email, so that a careless developer doesn't just leave this email in to get picked up by spam bots or similar)